rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/asset_search/action.py

```diff
@@ -0,0 +1,72 @@
+import ast
+
+import insightconnect_plugin_runtime
+from .schema import AssetSearchInput, AssetSearchOutput, Input, Output, Component
+
+
+# Custom imports below
+
+
+def format_response(resources: [], params={}):
+    assets = list()
+    ip = params.get(Input.IP)
+    hostname = params.get(Input.HOSTNAME)
+    for page in range(len(resources)):
+        current = str(resources[page])
+        asset = current.split("]},")
+        for num in range(len(asset)):
+            curr = asset[num]
```

Comment:
```suggestion
        for asset in assets:
```

Reply:
Replace `curr` with `asset`

Reply:
I need to use the num later because the first and last asset need to be handled differently so I can't do "for asset in assets:". But I will change the names.